node/core/provisioner/src/disputes/prioritized_selection/mod.rs

```diff
@@ -55,8 +55,8 @@ pub const MAX_DISPUTE_VOTES_FORWARDED_TO_RUNTIME: usize = 200;
 /// it's hard to predict this number because we can't guess how many new votes (for the runtime) a
 /// batch will contain.
 ///
-/// The value below is reached by: `MAX_DISPUTE_VOTES_FORWARDED_TO_RUNTIME` / 2 + 10%
-/// The 10% makes approximately means '10% new votes'. Tweak this if provisioner makes excessive
+/// The value below is calculated by: `MAX_DISPUTE_VOTES_FORWARDED_TO_RUNTIME` / 2 * 1.1%
+/// The `* 1.1%` approximately means 1.1% new votes. Tweak this if provisioner makes excessive
```

Comment:
@tdimitrov not quite sure about these numbers and the meaning of `/ 2`. Did you mean something else?

Reply:
The explanation is pointless and on top of it the comment itself is wrong.

Reply:
I've pushed the rewording too.